sklearn/metrics/base.py

```diff
@@ -131,3 +131,64 @@ def _average_binary_score(binary_metric, y_true, y_score, average,
         return np.average(score, weights=average_weight)
     else:
         return score
+
+
+def _average_multiclass_ovo_score(binary_metric, y_true, y_score, average):
+    """Uses the binary metric for one-vs-one multiclass classification,
+    where the score is computed according to the Hand & Till (2001) algorithm.
+
+    Parameters
+    ----------
+    y_true : array, shape = [n_samples]
+        True multiclass labels.
+        Currently only handles labels with values 0 to n_classes - 1.
```

Comment:
I would say "Assumes labels have been recoded to ``0 to n_classes``". Right now it sounds like this is a missing feature, but this is just an inner function where the work has already been done before.